### PR TITLE
targetp: new package

### PR DESCRIPTION
**Pull request by akthoma**
adding new package targetp

### CODE REVIEW

**Comment by akthoma** [comment]
bump

File: var/spack/repos/builtin/packages/targetp/package.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    """TargetP predicts the subcellular location of eukaryotic protein sequences.
+
+       Note: A manual download is required for TargetP.
+       Spack will search your current directory for the download file.
+       Alternatively, add this file to a mirror so that Spack can find it.
+       For instructions on how to set up a mirror, see
+       http://spack.readthedocs.io/en/latest/mirrors.html"""
+
+    homepage = "http://www.cbs.dtu.dk/services/TargetP/"
+    url      = "file://{0}/targetp-1.1b.Linux.tar.gz".format(os.getcwd())
```
Comment (junghans) [inline var/spack/repos/builtin/packages/targetp/package.py]: @scheibelp : Is there a better way to do this?

Comment (scheibelp) [reply var/spack/repos/builtin/packages/targetp/package.py]: I was meaning to check into that. It looks like we already have examples of this (e.g. gaussian) so LGTM.

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**